ci/ansible/roles/pulp-backup/tasks/main.yaml

```diff
@@ -0,0 +1,59 @@
+# rhel6 pulp 2.11.z backup
+- name: Stop services before backup
+  service: "name={{ item }} state=stopped"
+  with_items:
+    - httpd
+    - pulp_celerybeat
+    - pulp_resource_manager
+    - pulp_streamer
+    - pulp_workers
+    - qpidd
+  notify: Start Pulp
+
+- name: Install dependencies
+  action: "{{ ansible_pkg_mgr }} name={{ item }} state=present"
+  with_items:
+    - tar
+    - bzip2
+    - bzip2-libs
+
+- name: Create backups folder
+  file: path={{ remote_bkup_dir }} state=directory
+
+- name: Create local backups folder
+  local_action: file path={{ local_bkup_dir }} state=directory
+  become: false
+  when: fetch_to_localhost
```

Comment:
Got rid of the hacky file mode that I had here and use "become: false" so that the user that executed the ansible-playbook is used.